drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -141,9 +146,10 @@ static int wifi_nrf_umac_info(struct wifi_nrf_ctx_zep *rpu_ctx_zep)
 
 #ifndef CONFIG_NRF700X_RADIO_TEST
 	if (umac_info->mac_address0[0] == 0xffffffff &&
-	    umac_info->mac_address0[1] == 0xffffffff) {
-		LOG_ERR("Invalid MAC address0. OTP uninitialized !\n");
-		return -1;
+		umac_info->mac_address0[1] == 0xffffffff) {
```

Comment:
I have a fix to check for valid MAC address apart from 0xFFFFFFFF, may be we can add that fix as a separate commit in this PR itself.